src/components/Nav/Nav.spec.js

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import Nav from '.'
+
+describe('Nav', () => {
+  it('it renders', () => {
+    render(<Nav />)
+
+    const nav = screen.getByTestId('navButton')
```

Comment:
Why do you need to get it via `testId` this should be our _last_ choice. Usually we get the button via `getByRole`.

Reply:
i changed with getAllByRole.